### PR TITLE
Changes requested

### PR DESCRIPTION
**Pull request by WRRicht3r**

### CODE REVIEW

File: src/polkaswap-connect-wallet.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ app.
 
 2. You will see a list of supported wallet options to connect,
    including an account created on [Google](/polkaswap-connect-with-google.md), [Fearless Wallet](https://fearlesswallet.io),
-   Polkadot.js, Subwallet, and Talisman, as well as Wallet Connect.
+   Polkadot.js, as well as Wallet Connect for other compatible wallets.
```
Comment (stefashkaa) [inline src/polkaswap-connect-wallet.md]: - Google (simpler)
- Fearless (recommended)
- Polkadot.js
- Any substrate wallet installed locally
- WalletConnect with compatible wallets

Comment (WRRicht3r) [reply src/polkaswap-connect-wallet.md]: Updated with link to Polkadot wiki

File: .vitepress/config.mts [hunk 2]
```diff
@@ -173,6 +173,7 @@ export default withMermaid(
                 { text: "Create an Address", link: "create-an-address" },
                 { text: "Account Security", link: "account-security" },
                 { text: "Connect Wallet", link: "polkaswap-connect-wallet" },
+		{ text: "Connect with Google", link: "polkaswap-connect-with-google" },
```
Comment (stefashkaa) [inline .vitepress/config.mts]: fix spacing please

File: .vitepress/config.mts [hunk 3]
```diff
@@ -302,6 +303,7 @@ export default withMermaid(
                           text: "Connect Wallet",
                           link: "polkaswap-connect-wallet",
                         },
+			{ text: "Connect with Google", link: "polkaswap-connect-with-google" },
```
Comment (stefashkaa) [inline .vitepress/config.mts]: fix spacing please

File: .vitepress/config.mts (hunk 2 above)
Comment (stefashkaa) [inline .vitepress/config.mts]:
```suggestion
		        { text: "Connect with Google", link: "polkaswap-connect-with-google" },
```

File: .vitepress/config.mts [hunk 4]
```diff
@@ -298,14 +299,9 @@ export default withMermaid(
...
-                        {
-                          text: "Connect Wallet",
-                          link: "polkaswap-connect-wallet",
-                        },
-                        {
-                          text: "Send & Receive",
-                          link: "send-and-receive-polkaswap",
-                        },
+                        { text: "Connect Wallet", link: "polkaswap-connect-wallet", },
+			{ text: "Connect with Google", link: "polkaswap-connect-with-google" },
```
Comment (stefashkaa) [inline .vitepress/config.mts]:
```suggestion
			            { text: "Connect with Google", link: "polkaswap-connect-with-google" },
```

File: .vitepress/config.mts [hunk 5]
```diff
@@ -298,14 +299,9 @@ export default withMermaid(
...
+			{ text: "Send & Receive", link: "send-and-receive-polkaswap", },
```
Comment (stefashkaa) [inline .vitepress/config.mts]:
```suggestion
			            { text: "Send & Receive", link: "send-and-receive-polkaswap", },
```

**Review by stefashkaa — DISMISSED** [review]

**Review by stefashkaa — APPROVED** [review]

**Pull request merged**